### PR TITLE
Update community example links

### LINKED ISSUE #7083: PyTorch Geometric Link Broken In Docs
## 🐛 Bug

The link on this page doesn't resolve 

https://pytorch-lightning.readthedocs.io/en/latest/ecosystem/pytorch_ecoystem.html

We should consider highlighting other community ecosystem projects such as 
- https://github.com/manujosephv/pytorch_tabular
- https://github.com/asteroid-team/asteroid

### PR DESCRIPTION
**Pull request by awaelchli**
## What does this PR do?

Fixes #7083

## Before submitting
- [ ] Was this discussed/approved via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your PR does only one thing, instead of bundling different changes together?
- [ ] Did you make sure to update the documentation with your changes? (if necessary)
- [ ] Did you write any new necessary tests? (not for typos and docs)
- [ ] Did you verify new and existing tests pass locally with your changes?
- [ ] Did you update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)? (not for typos, docs, test updates, or internal minor changes/refactorings)

## PR review
Anyone in the community is free to review the PR once the tests have passed.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:

 - [x] Is this pull request ready for review? (if not, please submit in draft mode)
 - [x] Check that all items from **Before submitting** are resolved
 - [x] Make sure the title is self-explanatory and the description concisely explains the PR
 - [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

File: docs/source/ecosystem/pytorch_ecoystem.rst [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Pytorch Ecosystem Examples
+PyTorch Ecosystem Examples
 ==========================
 
-- `Pytorch Geometric: Deep learning on Graphs and other irregular structures <https://github.com/PyTorchLightning/pytorch-lightning/tree/master/pl_examples/pytorch_ecosystem/pytorch_geometric/README.md>`_.
+- `PyTorch Geometric: Deep learning on graphs and other irregular structures <https://github.com/rusty1s/pytorch_geometric/tree/master/examples/pytorch_lightning>`_.
```
Comment (awaelchli) [inline docs/source/ecosystem/pytorch_ecoystem.rst:4]: @tchaton check is this good?
We also have Geometric in the community samples, should we merge these sections into one page?

Comment (tchaton) [reply docs/source/ecosystem/pytorch_ecoystem.rst]: Yes, just pushed to your branch.

**Review by tchaton — APPROVED** [review]
LGTM !

File: docs/source/ecosystem/pytorch_ecoystem.rst (hunk 1 above)
Comment (awaelchli) [reply docs/source/ecosystem/pytorch_ecoystem.rst]: thanks

**Review by carmocca — APPROVED** [review]

**Review by s-rog — APPROVED** [review]

**Review by akihironitta — APPROVED** [review]

**Review by Borda — APPROVED** [review]

**Pull request merged**